pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -412,20 +412,21 @@ func proxyMapToEnvVars(proxyConfig map[string]string) []corev1.EnvVar {
 	return envVars
 }
 
-func gracefulTerminationDurationFromConfig(operatorSpec *operatorv1.StaticPodOperatorSpec) (int, error) {
+func gracefulTerminationDurationFromConfig(config map[string]interface{}) (duration int, err error) {
+	defer func() {
```

Comment:
This looks wrong.  If you want a default value, how about returning it in the case you want it used.